apps/molgenis-components/src/components/forms/RowEdit.vue

```diff
@@ -90,9 +90,11 @@ export default {
   },
   computed: {
     columnsWithoutMeta() {
-      return this.tableMetaData && this.tableMetaData.columns ? this.tableMetaData.columns.filter(
-        (column) => !column.name.startsWith("mg_")
-      ): [];
+      return this.tableMetaData && this.tableMetaData.columns
```

Comment:
return this.tableMetaData?.columns

Reply:
not really a fav ( yet ? ) but would have to be  this?.tableMetaData?.columns to replace pr code